chia/wallet/wallet_coin_store.py

```diff
@@ -126,17 +127,23 @@ async def set_spent(self, coin_name: bytes32, height: uint32):
 
         await self.add_coin_record(spent)
 
+    def coin_record_from_row(self, row: sqlite3.Row) -> WalletCoinRecord:
+        coin = Coin(bytes32(bytes.fromhex(row[6])), bytes32(bytes.fromhex(row[5])), uint64.from_bytes(row[7]))
+        return WalletCoinRecord(
+            coin, uint32(row[1]), uint32(row[2]), bool(row[3]), bool(row[4]), WalletType(row[8]), row[9]
```

Comment:
What is the type of row[9] - I notice it is the only one not explicitly cast

Reply:
a comment with the schema of this table would be very helpful, but I suppose this is not quite in scope for this PR